config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -104,7 +104,9 @@ default boolean isConsensusMigration() {
 
   Optional<Hash> getTerminalBlockHash();
 
-  List<Long> getForks();
+  List<Long> getForkBlockNumbers();
+
+  List<Long> getForkTimestamps();
```

Comment:
nit: feel like getForkNumbers or getForkBlockTimestamps would be more consistent since they both represent blocks, but happy to be convinced otherwise :)

Reply:
I like getForkBlockTimestamps. Changing it to that.